MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,566 @@
+browseItem.IsEnabled = false;
+
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.IsEnabled = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customize toolbar item view
+
+To Customize the toolbar items view by specifying their icon names and setting the view using `View` property in `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.save.jpg");
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[1];
+ImageEditorToolbarItem saveItem = zoomGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.View = image;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add SubToolbar
+
+You can create `SubToolbars` within your toolbar items to organize related actions.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image4.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem cropItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Crop");
+cropItem.SubToolbarOverlay = false;
+cropItem.SubToolbars = new List<ImageEditorToolbar>()
+{
+    new ImageEditorToolbar()
+    {
+        ToolbarItems = new List<IImageEditorToolbarItem>()
+        {
+           new ImageEditorToolbarItem(){ Name = "Rectangle"},
+           new ImageEditorToolbarItem(){ Name = "Pen"},
+           new ImageEditorToolbarItem(){ Name = "AddText"}
+        },
+    }
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add an item with the existing toolbar.
+
+You can explicitly add the `ImageEditorToolbarItem` to the `ImageEditorToolbar` manually.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.share.png");
+image.WidthRequest = 48;
+image.HeightRequest = 48;
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
+saveGroup.Items.Add(new ImageEditorToolbarItem() { Name = "Share", View = image });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Orientation
+
+The toolbar orientation can be changed to achieve the desired design by using the `Orientation` property in the `ImageEditorToolbar` class. The default value is `ToolbarOrientation.Horizontal`.
+
+For example, you can set the toolbar to a vertical orientation:
```

Comment:
For example, set the toolbar to a vertical orientation:

Reply:
Updated